dbusmock/testcase.py

```diff
@@ -41,81 +297,57 @@ class DBusTestCase(unittest.TestCase):
     session_bus_pid = None
     system_bus_pid = None
     _DBusTestCase__datadir = None
+    _busses: Dict[BusType, MockedDBus] = {
+          BusType.SESSION: None, # type: ignore
+          BusType.SYSTEM: None, # type: ignore
+    }
+
+    @staticmethod
+    def _bus(bustype: BusType) -> MockedDBus:
+        '''
+        Return (and create if necessary) the singleton DBus for the given bus type.
+        '''
+        if not DBusTestCase._busses.get(bustype):
+            DBusTestCase._busses[bustype] = MockedDBus(bustype)
+        return DBusTestCase._busses.get(bustype)  # type: ignore
```

Comment:
This can be a normal `[bustype]` instead of .get(), as it really ought to exist at this point.